.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["latest"]
```

Comment:
add to presets `"react"` and install `babel-preset-react` for development dependencies